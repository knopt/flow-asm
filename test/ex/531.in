4
6
-9.205174104909442
-5.911144945017439 8.615917804603885 3.17981594625042 -3.6355191619527627 
6.640470715614914 -7.583076327195566 5.097126814704154 4.530489279980724 
-1.3484433588683125 -4.92683859535211 -9.070416995753199 1.9688753351344346 
5.841040602221028 3.9323308374165844 0.16476725374590728 5.015787971584636 
-6.368197617032408 1.521133056670319 -5.760368487523591 -4.497475411070797 
-1.8987160768025664 9.998434094592653 9.647157448285132 2.969726077944907 

-5.543591222820636 2.929116312795891 -8.00588099679322 4.143439449887625 9.765235579442269 -0.9713494191989192 
