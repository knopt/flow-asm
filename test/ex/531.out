-206.21049241583273 309.638955344613 34.564668780188164 -221.92692147808683 
476.8619155401497 -384.8602967569773 329.76575531339887 235.69601736903903 
-169.71478630041918 -300.5649549183278 -396.02275136448344 39.77104906297563 
291.3767960019932 233.53662124461167 139.4206084645629 394.15125089960935 
-454.8501549409869 -34.371926275403524 -503.444229091519 -322.3010238618388 
-76.66163762961814 348.2064991847842 223.04402283061793 90.60173795066896 
